ang/civicase/CaseDetails--placeholder.html

```diff
@@ -37,5 +37,41 @@
       </div>
     </div>
     <!-- End - Subject and QuickLinks -->
+    <!-- Next Milestone -->
```

Comment:
Based on the comment on `CaseDetails--tabs--summary.html`, this needs to change aswell.